news/2019-11-12-winter-2019-fanart-contest.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Winter 2019 Fanart Contest"
+date: 2019-11-12 09:45:00 +0000
+---
+
+The end of the year is rapidly approaching, and while things might be cooling down where you are (or maybe not), the artist's battlefield is just as fiery as ever!
+
+![](https://assets.ppy.sh/contests/82/header.jpg)
+
+Do you like to draw? Great, because we've got just the contest for you!
+
+**Draw any of the osu! mascots (official or community) in a winter-themed piece** and you'll have a shot at getting your work showcased in the game's main menu as a part of our Winter 2019 Seasonal Background pack!
+
+The hard-fought finalists will also nab themselves a few months of **osu!supporter** (the exact amount yet to be determined), and the top three finalists as voted by YOU will receive **an extra shiny profile badge** showcasing their work!
+
+"But Ephemeral," you might ask, "these are *new* prizes, what gives?" What gives is the season of giving, and that's precisely what this time of year tends to be for most of the world. That and most importantly, we've heard your feedback on the prizes for the winning artists crystal clear. *(Yes, previous fanart contest top 3 finalists will eventually receive profile badges for their winning entries as well. You won't be left out!)*
```

Comment:
don't really like the tone of this, maybe don't mention your own name.

Reply:
changed the tone